pkg/fanal/image/image.go

```diff
@@ -128,3 +129,31 @@ func LayerIDs(img v1.Image) ([]string, error) {
 	}
 	return layerIDs, nil
 }
+
+func GuessBaseImageIndex(histories []v1.History) int {
```

Comment:
I think this comment should be moved to this function as the main logic exists here.
https://github.com/aquasecurity/trivy/blob/87844ca3506ab8907d53ce0e6cb69f9af0fdbc1b/pkg/fanal/artifact/image/image.go#L473-L501

Reply:
Maybe keep the first line only in Artifact.guessBaseLayers().

Reply:
You are right. Made this in https://github.com/aquasecurity/trivy/pull/4344/commits/d8c4002f656d6ad32b7f2796d4e43686fc42a161